scripts/nginx-ubi/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
We should remove all reference to Kabanero from this file.

Reply:
If we are not mentioning Kabanero then we will have to use something and we will need to pass this in as a variable. We will also need to use the same variable at other places in the hub_build.sh script.

Reply:
Done